interfaces/matlab_experimental/Setup/buildToolbox.m

```diff
@@ -0,0 +1,104 @@
+function buildToolbox()
```

Comment:
Can we pass arguments to this function rather than reading environment variables?